typescript/packages/plugins/spl-token/src/utils/getTools.ts

```diff
@@ -0,0 +1,36 @@
+import type { DeferredTool, SolanaWalletClient } from "@goat-sdk/core";
+import type { NetworkSpecificToken } from "../tokens";
+import { getBalanceParametersSchema, transferParametersSchema } from "../parameters";
+import type { Connection } from "@solana/web3.js";
+import type { z } from "zod";
+import { transfer } from "../methods/transfer";
+import { balanceOf } from "../methods/balance";
+
+export function getTools(
+    tokenList: NetworkSpecificToken[],
+    connection: Connection,
+): DeferredTool<SolanaWalletClient>[] {
+    const tools: DeferredTool<SolanaWalletClient>[] = [];
+
+    for (const token of tokenList) {
```

Comment:
I think it may make sense to have a tool for getting the balance / transfer an arbitrary token? By token address for ex for now

Reply:
Right now this wont work w memecoins, i think that wouldbe an issue